tests/plugins/notification/leakglobalscope.js

```diff
@@ -0,0 +1,20 @@
+/* bender-tags: editor, 1057, notification */
+/* bender-ckeditor-plugins: notification */
+
+( function() {
+	'use strict';
+
+	bender.editor = true;
+
+	bender.test( {
+		'test Notification should not leak the global scope': function() {
+			// Web Notification Api is not supported by IE.
+			if ( CKEDITOR.env.ie ) {
```

Comment:
This assert omits also Edge, in which Web Notifcations API is implemented.

What's more, leakage would also be present in old IE. It rather check if `window.Notification` is undefined or a function with correct body.